frontend/javascripts/oxalis/shaders/main_data_shaders.glsl.ts

```diff
@@ -202,7 +203,8 @@ void main() {
 
   // Get Color Value(s)
   vec3 color_value  = vec3(0.0);
-  <% _.each(colorLayerNames, function(name, layerIndex) { %>
+  <% _.each(orderedColorLayerNames, function(name, layerIndex) { %>
```

Comment:
do you think it's possible to generate the shader code so that it doesn't need to be recompiled when the layer order changes? on my machine, the shader compilation takes a few seconds which means that the UI is blocked everytime I change the layer order :S
`orderedColorLayerNames` can probably be passed as a uniform, but I'm not sure what needs to be done additionally (if it's possible at all).

Reply:
I discussed this with Tom and we think that this shouldn't block the PR as the reordering won't be used with high frequency at the moment. Maybe we revamp the rendering architecture to be multi-pass soon. Then, this issue can be solved easier probably.